jplag.endToEndTesting/src/main/java/de/jplag/end_to_end_testing/constants/Constant.java

```diff
@@ -0,0 +1,32 @@
+package de.jplag.end_to_end_testing.constants;
+
+import java.nio.file.Path;
+
+/**
+ * All constant values that are needed in the test cases or helper classes.
+ */
+public final class Constant {
```

Comment:
Name too generic, use something like `TestDirectoryConstants` or similar (also, plural).

Reply:
Ok i'll change the class name and also change the logger functions to Debug.